.github/workflows/ci.yml

```diff
@@ -21,6 +21,7 @@ jobs:
           node-version: 18
       - run: npm ci
 
+      - run: ./node_modules/.bin/prettier --check .
```

Comment:
Please use `npx prettier . --check` or `npm run prettier` with a custom script in package.json instead

Reply:
Done in d3b6f054569686ce2132b7ba545fb4c348897233